miio/version.py

```diff
@@ -1,2 +1,2 @@
 # flake8: noqa
-__version__ = "0.4.8"
+__version__ = "0.5.0"
```

Comment:
Please do not bump the version in feature PRs.

Reply:
Will keep in mind. I need this to make HA install this fork correctly

Reply:
Ok, could you please remove this and update the README to contain information about miot support & the new devices? I think we can merge and prepare a new release then :tada:

Reply:
:+1: